tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Service.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Service Platform.
+
+You can follow this tutorial to configure any NServiceBus based solution for monitoring with the Particular Service Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
```

Comment:
Should we use `system` instead of `solution`? For me solution sounds pretty VS centric.